migrations/20181114153441_ammended_section_intro_column_types.js

```diff
@@ -0,0 +1,20 @@
+const { noop } = require("lodash");
+
+exports.up = async function(knex) {
+  await knex.raw(`DROP VIEW "SectionsView"`);
+  await knex.schema.table("Sections", table => {
+    table.text("introductionTitle").alter();
+    table.text("introductionContent").alter();
+  });
+  return knex.raw(`
```

Comment:
Why do we need to recreate the view?

Reply:
We cant amend a table without dropping and recreating the view